pkg/agent/openflow/pipeline.go

```diff
@@ -269,7 +267,7 @@ func (c *client) reEntranceBypassCTFlow(gwPort, reentPort uint32, category cooki
 	conntrackCommitTable := c.pipeline[conntrackCommitTable]
 	return conntrackCommitTable.BuildFlow(priorityHigh).MatchProtocol(binding.ProtocolIP).
 		MatchRegRange(int(marksReg), portFoundMark, ofPortMarkRange).
-		MatchInPort(gwPort).MatchRegRange(int(portCacheReg), reentPort, ofPortRegRange).
+		MatchInPort(gwPort).MatchReg(int(portCacheReg), reentPort).
```

Comment:
@wenyingd : I changed the ofport match, because now we are using all bits of reg1, and the realized match will not use the matchRegRange format. Let me know if you think we should just use matchRegRange.

Reply:
I am OK to use `MatchReg` instead of `MatchRegRange` when matching ofport number.